### PR TITLE
root: add v6.30.08, v6.32.02

### PR DESCRIPTION
**Pull request by wdconinc**
This PR adds two new bugfix releases for `root`, in the 6.30 and 6.32 cycles. No changes to the spack package needed per diffs at:
- https://github.com/root-project/root/compare/v6-32-00...v6-32-02
- https://github.com/root-project/root/compare/v6-30-06...v6-30-08

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Verified Checksums. Looks good to me. Waiting on final approval from @vitodb, @vvolkl, @HadrienG2, @marcmengel, @drbenmorgan, @greenc-FNAL, @gartung.

**Review by marcmengel — APPROVED** [review]

**Pull request merged**
